### ISSUE TITLE
Failed to activate the northem-light-atom-syntax theme

### ISSUE THREAD

**Issue by paulidealiste** [arcticicestudio/northem-light-atom-syntax#1]
This error occurs when the following steps are taken:
1. Install northem-light-atom-syntax theme 0.4.0
2. Try to change to this syntax theme from the Atom settings as usual

**Atom Version**: 1.0.5
**System**: Microsoft Windows 10 Pro
**Thrown From**: [northem-light-atom-syntax](https://github.com/arcticicestudio/northem-light-atom-syntax) package, v0.4.0
### Stack Trace

Failed to activate the northem-light-atom-syntax theme

```
At variable @dirty-white is undefined in C:\Users\pauli\.atom\packages\northem-light-atom-syntax\styles\colors.less:130:34

LessError: variable @dirty-white is undefined
  at C:\Users\pauli\.atom\packages\northem-light-atom-syntax\styles\colors.less:130:34
```
### Commands

```
     -0:11.6.0 settings-view:uninstall-themes (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "northem-light-atom-ui",
      "northem-light-atom-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
color-picker, v2.0.11
file-icons, v1.6.3
linter, v1.3.4
linter-jshint, v1.1.5
northem-light-atom-ui, v0.1.0

# Dev
No dev packages
```

**Comment by arcticicestudio** [comment]
Thanks for reporting!
Released a fixed apm package update [`0.4.1`](https://github.com/arcticicestudio/northem-light-atom-syntax/releases/tag/v0.4.1) :smile:

**Comment by paulidealiste** [comment]
Great stuff. It`s after all my favorite Atom theme. Keep up the good work.
